docs/source/installation.rst

```diff
@@ -83,127 +83,81 @@ Note: not all soft dependencies of ``sktime`` are also available on ``conda-forg
 The other soft dependencies can be installed via ``pip``, after ``conda install pip``.
 
 
-Development versions
---------------------
-To install the latest development version of ``sktime``, or earlier versions, the sequence of steps is as follows:
+Development version install
+---------------------------
 
-| Step 1 - ``git`` clone the ``sktime`` repository, the latest version or an earlier version.
-| Step 2 - ensure build requirements are satisfied
-| Step 3 - ``pip`` install the package from a ``git`` clone, with the ``editable`` parameter.
+Development versions of ``sktime`` are installs from static snapshots of the ``sktime`` repository.
 
-Detail instructions for all steps are given below.
-For brevity, we discuss steps 1 and 3 first; step 2 is discussed at the end, as it will depend on the operating system.
+Installing development versions is useful for testing, but not recommended
+for development and contribution. For development and contribution, see the next section
+on full developer setup.
 
-Step 1 - clone the git repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The ``sktime`` repository should be cloned to a local directory, using a graphical user interface, or the command line.
-
-Using the ``git`` command line, the sequence of commands to install the latest version is as follows:
+To install the latest development version of ``sktime``,
+you can use the ``pip`` package manager to install directly from the GitHub repository:
 
 .. code-block:: bash
 
-    git clone https://github.com/sktime/sktime.git
-    cd sktime
-    git checkout main
-    git pull
+    pip install git+https://github.com/sktime/sktime.git
 
 
-To build a previous version, replace line 3 with:
+To install from a specific branch, use the following command:
 
 .. code-block:: bash
 
-    git checkout <VERSION>
-
-This will checkout the code for the version ``<VERSION>``, where ``<VERSION>`` is a valid version string.
-Valid version strings are the repository's ``git`` tags, which can be inspected by running ``git tag``.
-
-You can also `download <https://github.com/sktime/sktime/releases>`_ a zip archive of the version from GitHub.
+    pip install git+https://github.com/sktime/sktime.git@<branch_name>
```

Comment:
Same as above, it may be useful to mention how to install optional dependencies.